drake/examples/TwoWheeledInvertedPendulum/TWIP.m

```diff
@@ -0,0 +1,145 @@
+classdef TWIP < DrakeSystem 
+  
```

Comment:
add a description (like what you have here:
The dynamics of the two wheeled inverted pendulum are similar to the cartpole, but different in the sense that the torque provided to the wheels by the motor also induces a counter-torque on the body. This makes it more difficult to stabilize and swing-up. )